commons/pom.xml

```diff
@@ -17,6 +17,7 @@
       <dependency>
          <groupId>com.github.ben-manes.caffeine</groupId>
          <artifactId>caffeine</artifactId>
+         <optional>true</optional>
```

Comment:
Can you clarify this. Would `infinispan-commons` work fine when caffeine is missing?